packages/frontend/package.json

```diff
@@ -115,29 +111,28 @@
         "@types/canvas-confetti": "^1.6.0",
         "@types/isomorphic-fetch": "^0.0.39",
         "@types/lodash": "^4.14.202",
-        "@types/react": "^18.3.5",
-        "@types/react-dom": "^18.3.0",
+        "@types/react": "19.0.2",
+        "@types/react-dom": "19.0.2",
         "@types/react-grid-layout": "^1.3.1",
-        "@types/react-helmet": "^6.1.6",
         "@types/react-syntax-highlighter": "^13.5.2",
-        "@types/styled-components": "^5.1.22",
+        "@types/styled-components": "^5.1.34",
         "@types/uuid": "^8.3.4",
-        "@vitejs/plugin-react": "^4.3.1",
+        "@vitejs/plugin-react": "^4.3.4",
         "eslint-plugin-css-modules": "^2.11.0",
         "eslint-plugin-jest-dom": "^5.1.0",
         "eslint-plugin-jsx-a11y": "^6.6.1",
-        "eslint-plugin-react": "^7.31.8",
-        "eslint-plugin-react-hooks": "^4.6.0",
+        "eslint-plugin-react": "7.32.2",
```

Comment:
to fix: https://github.com/es-shims/iterator-helpers/issues/4

no need to add anothe resolution, this version fixes it.